src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -17,31 +19,73 @@
  *
  */
 public class Drive extends DifferentialDrive {
-	// TODO: DEfine logger
-
+	private ControlMode controlMode;
+	// TODO: DEfine logger (Done)
+	private static final Logger LOGGER = Logger.getLogger(Drive.class);
 
 	// Single instance of this class
 	private static Drive instance = null;
 
 	// Data storage object
 	private DataStorage data;
 	
-	public static WPI_TalonSRX left = new WPI_TalonSRX(1);
-	public static WPI_TalonSRX right = new WPI_TalonSRX(2);
+	private WPI_TalonSRX leftFollower1;
+	private WPI_TalonSRX leftFollower2;
+	private WPI_TalonSRX rightFollower1;
+	private WPI_TalonSRX rightFollower2;
+	
+	private WPI_TalonSRX leftLead;
+	private WPI_TalonSRX rightLead;
 
 	// Private constructor
-	private Drive() {
-		super(left, right); // Need to specify the motor channels.
+	private Drive(WPI_TalonSRX left, WPI_TalonSRX leftFollower1, WPI_TalonSRX leftFollower2,
+			WPI_TalonSRX right, WPI_TalonSRX rightFollower1, WPI_TalonSRX rightFollower2) {
+		super(left, right);
+		
+		
+		// Need to specify the motor channels.
+		// TODO: Define and initialize motors. (Done)
+		
+		this.leftLead = left;
+		initMotor(this.leftLead);
+
+		this.leftFollower1 = leftFollower1;
+		initMotor(this.leftFollower1);
+		initMotorForFollowerMode(left, leftFollower1);
+
+		this.leftFollower2 = leftFollower2;
+		initMotor(this.leftFollower2);
+		initMotorForFollowerMode(left, leftFollower2);
+		
+		this.rightLead = right;
+		initMotor(this.rightLead);
+
+		this.rightFollower1 = rightFollower1;
+		initMotor(this.rightFollower1);
+		initMotorForFollowerMode(right, rightFollower1);
+		
+		this.rightFollower2 = rightFollower2;
+		initMotor(this.rightFollower2);
+		initMotorForFollowerMode(right, rightFollower2);
+		
+	// Make objects
+	data = DataStorage.getInstance();
+	
 
-		// TODO: Define and initialize motors.
+}
+	
+	
+	
 
-		// Make objects
-		data = DataStorage.getInstance();
+		
 
-	}
 	
 	private void initMotor(WPI_TalonSRX talon) {
-		// TODO: Set the default Talon parameters
+		talon.set(ControlMode.PercentOutput, 0);
+		talon.selectProfileSlot(RobotMap.VELOCITY_PID_PROFILE, 0);
+		talon.configAllowableClosedloopError(0, RobotMap.VELOCITY_ALLOWABLE_CLOSED_LOOP_ERROR, 0);
+		talon.configNominalOutputReverse(-1.0, 1);
```

Comment:
Not so simple. There are a few other config methods for configuring voltage levels.